dino.icu.yaml

```
@@ -417,3 +417,8 @@ xn--ct9h:
 xn--rl8h:
   type: CNAME
   value: maxwofford.github.io.
+
+node:
+- ttl: 1
```

```suggestion
- ttl: 600
```

TTL needs to be bumped to at least 600